### PR TITLE
Steup class refactoring

### PR DESCRIPTION
**Pull request by DougWelchons**
Katy, please take a look at these and double check our work.

I did remove the player board being passed into the player, because its unneeded.

### CODE REVIEW

File: lib/setup.rb [hunk 1]
```diff
@@ -6,120 +6,69 @@
...
 
-  def render_player_board
-    # @player.collect_ships
-
+  def prompt_player
     puts "(Computer): I have laid out my ships in the grid."
     puts "            You now need to lay out your two ships."
     puts "            The Cruiser is three units long and the"
     puts "            Submarine is two units long."
+    sleep(2)
```
Comment (klatour324) [inline lib/setup.rb:39]: Cool implementation of sleep() here! I like that it gives a slight pause to displaying the rendered board, so a person has time to read the computer's message about what they should do next.

File: test/setup_test.rb [hunk 2]
```diff
@@ -12,6 +12,7 @@ def test_it_exists
   end
 
   def test_it_can_collect_ships
+    skip
```
Comment (klatour324) [inline test/setup_test.rb]: We don't need this test anymore correct since we got rid of the collect_ships method?

File: lib/setup.rb [hunk 3]
```diff
@@ -6,120 +6,69 @@
...
 
 class Setup
-  attr_reader :computer,
+  attr_reader :ships,
               :player,
-              :computer_board,
-              :player_board,
-              :ships
-
+              :computer
```
Comment (klatour324) [inline lib/setup.rb:11]: Is this supposed to be an attr reader?

**Comment by klatour324** [comment]
Hey Doug,

Check out my comments. Let me know  if you have any questions, otherwise we should be close to merging in this branch!

File: lib/setup.rb (hunk 3 above)
Comment (DougWelchons) [reply lib/setup.rb:11]: I think the GitHub rendering makes it a bit confusing whats going on here. but i double checked the code, and it looks correct. this is what we have

Screen Shot 2020-12-15 at 1 24 32 PM

File: test/setup_test.rb (hunk 2 above)
Comment (DougWelchons) [reply test/setup_test.rb]: Good catch. I went back and removed it.

File: lib/setup.rb (hunk 3 above)
Comment (klatour324) [reply lib/setup.rb:11]: Ah! Okay, thank you for the clarification here. Yes, paying attention to the line numbers on Git Hub is important. I appreciate the screen share just to confirm!

**Comment by klatour324** [comment]
Looks good Doug! I'll merge.

**Pull request merged**